source/javascripts/refills/coffeescript/expander.coffee

```diff
@@ -1,9 +1,7 @@
 $(document).ready ->
-  expanderTrigger = document.getElementById("js-expander-trigger")
-  expanderContent = document.getElementById("js-expander-content")
-  $("#js-expander-trigger").click ->
-    $(this).toggleClass "expander-hidden"
+  expanderTrigger = $('.expander-trigger')
+  expanderContent = $('.expander-content')
```

Comment:
Same issue that Hound found below. Looks like we aren't actually using `expanderContent`